### DISCUSSION TITLE
Web server not working

### DISCUSSION THREAD

**Discussion by vftable** [JohnnyCrazy/SpotifyAPI-NET#651] [Q&A]
Hey, so when I was trying to generate an authorization web server, I navigated to the callback url and it failed to connect. In my Visual studio console it says that "The thread - exited with code 0 (0x0)." Any idea on how to fix it???

**Comment by JohnnyCrazy** [comment]
Hi,

Sorry for the late response. Did you start the webserver according to the examples? I would need some more code to see where the problem is

**Reply by vftable** [reply]
Yeah, I did. I tried to go to localhost, and then it did nothing. It just kept loading, and in the program it had said the above code
It was the authorization code with web server example with no edits
